internal/metrics/reporter_test.go

```diff
@@ -19,21 +19,50 @@ import (
 var _ = Describe("Reporter", func() {
 	var reporter *metrics.Reporter
 	var ingestRequest metricspb.IngestRequest
+	var okServer *httptest.Server
+	var okURL string
+	var retryableServer *httptest.Server
+	var retryableURL string
+	var unretryableServer *httptest.Server
+	var unretryableURL string
 
-	JustBeforeEach(func() {
+	BeforeSuite(func() {
```

Comment:
`BeforeSuite` is another weird one. A "suite" is a whole package's tests, rather than a `Describe` block's tests. Consequently, you can only have one `BeforeSuite` per package, and it's almost always at the topmost level.